_posts/2020-09-24-Using-Zeek-to-find-persistent-threats-by-monitoring-DNS.md

```diff
@@ -0,0 +1,222 @@
+---
```

Comment:
Run file through prettier, command is in README.md. Then its nicely readable also in editor.